Zend/Optimizer/zend_inference.c

```diff
@@ -3885,6 +3885,7 @@ static zend_always_inline zend_result _zend_update_type_info(
 		case ZEND_DO_ICALL:
 		case ZEND_DO_UCALL:
 		case ZEND_DO_FCALL_BY_NAME:
+		case ZEND_INIT_METHOD_CALL:
```

Comment:
This looks wrong. `ZEND_INIT_METHOD_CALL` does not define a value. Methods are called using `DO_FCALL`, so this is already handled.